wallet/src/repository/wallet/mod.rs

```diff
@@ -1001,6 +1002,66 @@ where
         self.create_transaction_components(state, value, fee, recipient, false)
     }
 
+    fn create_vt_body(
+        &self,
+        state: &mut State,
+        value: u64,
+        fee: u64,
+        recipient: Option<(types::PublicKeyHash, u64)>,
+    ) -> Result<(types::VTTransactionBody, types::TransactionComponents)> {
+        // Set this to 1-2 output.
+        let mut initial_fee = u64::from(INPUT_SIZE + 2 * OUTPUT_SIZE) * fee;
```

Comment:
Multipliying by fee can overflow if the fee is too large. In that case we should return an error, probably the user wanted to set 1_000_000 nanowits but it accidentally sent 1_000_000 wits, so the error should be FeeTooLarge or something similar.

Reply:
OK!